packages/widget_toolkit_biometrics/lib/src/base/models/biometrics_setting_message_types.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:widget_toolkit/language_picker.dart';
+
+enum BiometricsSettingMessageType {
+  biometricsAreDisabled,
+  noBiometricsAvailable,
+  biometricEnabledSuccess,
+  biometricDisabledSuccess,
+  noBiometricsSupportedDevice,
+  none,
+}
+
+extension ReadableMessage on BiometricsSettingMessageType {
+  String translate() {
```

Comment:
@PrimeVoxed / @RomanovaPrime please add documentation in the package README file how those messages can be localised